src/app/presensi-siswa/scan/page.tsx

```
@@ -0,0 +1,837 @@
+
+					ctx.fillStyle = "#ffffff";
+					ctx.textAlign = "center";
+					ctx.textBaseline = "middle";
+					ctx.fillText(text, badgeX + badgeWidth / 2, badgeY + badgeHeight / 2);
+				}
+			}
+		});
+	};
+
+	// RFID USB Reader Listener (Continuous background keystrokes)
+	useEffect(() => {
```

Biome (2.5.3)

[error] This hook does not specify its dependency on handleScanSubmit.

(lint/correctness/useExhaustiveDependencies)